packages/fxa-auth-server/test/local/senders/email.js

```diff
@@ -1337,7 +1337,7 @@ const TESTS = [
       ['X-Template-Version', { test: 'equal', expected: TEMPLATE_VERSIONS.cadReminderFirst }],
     ])],
     ['html', [
-      { test: 'include', expected: "Here&#x27;s your reminder to sync devices." },
+      { test: 'include', expected: "Here's your reminder to sync devices." },
```

Comment:
Again, jsdom encodes and parses the html string because of which this test was failing so I had to tweak it a bit

Reply:
Hmm, I don't know that we should need to touch these tests.  You created that `mjml-emails` file, which should run the tests for the MJML version of the templates. I think `email.js` should remain testing the old version of templates that are in production.